packages/ecr-credential-provider-1.32/ecr-credential-provider-1.32.spec

```diff
@@ -0,0 +1,77 @@
+%global goproject github.com/kubernetes
+%global gorepo cloud-provider-aws
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.32.0
+%global rpmver %{gover}
+
+%global _dwz_low_mem_die_limit 0
+
+Name: %{_cross_os}ecr-credential-provider-1.32
+Version: %{rpmver}
+Release: 0.beta1%{dist}
```

Comment:
It's weird to label this the "beta1" release when it's the actual upstream 1.32.0 release. If you needed to put out a revised version of this package while keeping the upstream release the same, you would increment the release field instead.

Reply:
Discussed offline: we should not enforce arbitrary versioning and we prefer to match the source. Will change this to the official release: `1%{dist}`